Library/Homebrew/utils/github.rb

```diff
@@ -269,6 +269,7 @@ def uri_escape(query)
 
   def issues_for_formula(name, options = {})
     tap = options[:tap] || CoreTap.instance
+    return [] if tap.custom_remote?
```

Comment:
I think I'd like if this did a little more inspection that this; can you instead see if the remote contains e.g. `/github.com/` and only `return []` if it's not possible to extract a github `full_name` from it?

Reply:
Was this the only objection?  I ask because well I wanted to setup up private taps against private repos and I ran into this

Reply:
@jmahowald At the time: yes. Please feel free to open a PR with this comment addressed. Thanks!